static/app/components/replays/breadcrumbs/replayTimeline.tsx

```diff
@@ -68,13 +68,13 @@ export default function ReplayTimeline() {
       >
         <MinorGridlines durationMs={durationMs} width={width} />
         <MajorGridlines durationMs={durationMs} width={width} />
+        <TimelineScrubber />
         <TimelineGaps
           durationMs={durationMs}
           frames={appFrames}
           totalFrames={chapterFrames.length}
           width={width}
         />
```

Comment:
This this re-ordering, just want to confirm that the Timeline can still be clicked on to jump to a specific timestamp? 

Can't click on the spots where the gray box is rendered, those eat clicks. But where no gray box is rendered clicks should still get handled by the TimelineScrubber

Reply:
Yes!